snap/local/validate_config.sh

```diff
@@ -58,4 +58,12 @@ for opt in tls include_hidden use_sim_time use_compression; do
   fi
 done
 
+# Enforce bash list as a string synthax
+for opt in topic-whitelist param-whitelist service-whitelist client-topic-whitelist; do
+  VALUE=$(snapctl get "${opt}")
+  if [ -n "${VALUE}" ]; then
+    snapctl set "${opt}"=$(echo ${VALUE} | tr -d ' ')
```

Comment:
Should this be a string just in case? `snapctl set "${opt}"="$(echo ${VALUE} | tr -d ' ')"`

Reply:
`tr` always returns a string

Reply:
I can add it it you want